include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp

```diff
@@ -276,7 +281,8 @@ class pairwise_alignment_algorithm_banded :
 
         for (auto && sequence2_value : sequence2 | views::drop(1))
         {
-            auto cell = *alignment_column_it;
+            last_alignment_column_it = alignment_column_it;
+            auto cell = *last_alignment_column_it;
```

Comment:
Can you explain this change to me? What are you tracking here since it seems `last_alignment_column_it` is always overridden with the current column iterator, so they are always the same no?

Reply:
Not quite. I think I will add a small example in the documentation. 
The problem is based on the internal implementation using a single column score matrix. As long as the band touches the first row of the matrix the relative column index points to the same cell inside of this column as the column before. When the band starts to move downwards the relative index to the previous column changes. This is reflected with the special function for the banded alignment. But before I try to explain it here I try to make a picture of it in the documentation.